src/main/resources/files/en/imprint

```diff
@@ -0,0 +1,23 @@
+Imprint Qabel GmbH
+
+Qabel GmbH
+Goseriede 4 / Tiedthof
+30159 Hannover
+Germany
+
+Register court: Amtsgericht Hannover, HRB 210757
+USt-IdNr.: DE293570022
+
+E-Mail: devnull@qabel.de (GPG)
```

Comment:
All these `(GPG)` suffixes don't make sense in plaintext. Either remove them or link them to the key or add a hash of the key in plaintext.